vocabularies/ABAP.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.01">
```

Comment:
`4.01`: have you checked with UI5 whether they accept this? All other vocabularies are `4.0`

Reply:
No, CodeList has 4.01.